File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem01_MergeStringsAlternatelyTest.java
package org.phoenix.leetcode.challenges;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem01_MergeStringsAlternatelyTest {

    private final Problem01_MergeStringsAlternately test = new Problem01_MergeStringsAlternately();

    @org.junit.jupiter.api.Test
    void mergeAlternately() {
        String word1 = "abc";
        String word2 = "pqr";
        assertEquals("apbqcr", test.mergeAlternately(word1, word2));

        word1 = "ab";
        word2 = "pqrs";
        assertEquals("apbqrs", test.mergeAlternately(word1, word2));
    }
}

File: JavaScript/leetcode75/src/test/Problem01_MergeStringsAlternately.test.js
const mergeAlternately = require('../main/Problem01_MergeStringsAlternately');

test('should output', () => {
    const word1 = 'abc', word2 = 'pqr';
    const res = mergeAlternately(word1, word2);
    expect(res).toBe('apbqcr')
})